### PR TITLE
Example with docker and ACI for C# and F#

### PR DESCRIPTION
**Pull request by mikhailshilkov**
Two examples that show `docker.Image` usage for .NET SDK

Depends on https://github.com/pulumi/pulumi-docker/pull/115 being available in NuGet

### CODE REVIEW

File: azure-cs-cosmosapp-component/Azure.CosmosAppComponent.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi.Azure" Version="1.4.0-preview" />
+    <PackageReference Include="Pulumi.Docker" Version="1.0.0" />
```
Comment (mikhailshilkov) [inline azure-cs-cosmosapp-component/Azure.CosmosAppComponent.csproj]: This is a fake version that I tested locally with. It needs to change before we merge.

**Comment by stack72** [comment]
@mikhailshilkov please can you rebase this when you get a chance :)

File: azure-cs-cosmosapp-component/Azure.CosmosAppComponent.csproj (hunk 1 above)
Comment (stack72) [reply azure-cs-cosmosapp-component/Azure.CosmosAppComponent.csproj]: @mikhailshilkov can you also update to the correct version of the Pulumi.Docker plugin as well :)

**Comment by mikhailshilkov** [comment]
@stack72 Done, sorry it took me so long

**Pull request merged**